### PR TITLE
Gql05 user delete by id ap itests kutovyi

### PR DESCRIPTION
**Pull request by rustamkutovyi**

### CODE REVIEW

File: src/modules/user/_test/userDeleteById.spec.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                variables: userCreateArg
+            }
+            gqlRequest(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err)
+                    const respData = res.body.data
+                    userId = respData.userCreate._id
+                    console.log(userId)
+                    console.log(respData)
```
Comment (itmelia) [inline src/modules/user/_test/userDeleteById.spec.js:25]: Please delete console.log from rows 24, 25

File: src/modules/user/_test/userDeleteById.spec.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                variables: userDeleteArg
+
+            }
+            gqlRequest(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err)
+                    const respData = res.body.data
+                    expect(respData.userDeleteById).to.eq(true)
+                    console.log(respData)
```
Comment (itmelia) [inline src/modules/user/_test/userDeleteById.spec.js:47]: Please delete console.log

File: src/modules/user/_test/userDeleteById.spec.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                variables: userDeleteArg
+
+            }
+            gqlRequest(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err)
+                    const respData = res.body.data
+                    expect(respData.userDeleteById).to.eq(false)
+                    console.log(respData)
```
Comment (itmelia) [inline src/modules/user/_test/userDeleteById.spec.js:69]: Please delete console.log

File: src/modules/user/_test/userDeleteById.spec.js [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                variables: userDeleteArg
+
+            }
+            gqlRequest(postData)
+                .expect(200)
+                .end((err, res) => {
+                    if (err) return done(err)
+                    const respData = res.body.data
+                    expect(respData.userDeleteById).to.eq(null)
+                    console.log(respData)
```
Comment (itmelia) [inline src/modules/user/_test/userDeleteById.spec.js:89]: Please delete console.log

File: src/modules/user/_test/userDeleteById.spec.js [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+            const userDeleteArg = {
+                userId: 34314534
+            }
+            const postData = {
+                query: userDeleteQ,
+                variables: userDeleteArg
+
+            }
+            gqlRequest(postData)
+                .expect(400)
```
Comment (itmelia) [inline src/modules/user/_test/userDeleteById.spec.js:104]: I guess we should expect 200. In this case, respData.userDeleteById equals null and we get an error message.

**Review by itmelia — COMMENTED** [review]
Rustam, please review my comments.

File: src/modules/user/_test/userDeleteById.spec.js (hunk 5 above)
Comment (rustamkutovyi) [reply src/modules/user/_test/userDeleteById.spec.js:104]: I think we have bug here

File: src/modules/user/_test/userDeleteById.spec.js (hunk 4 above)
Comment (rustamkutovyi) [reply src/modules/user/_test/userDeleteById.spec.js:89]: Done

File: src/modules/user/_test/userDeleteById.spec.js (hunk 3 above)
Comment (rustamkutovyi) [reply src/modules/user/_test/userDeleteById.spec.js:69]: Done

File: src/modules/user/_test/userDeleteById.spec.js (hunk 2 above)
Comment (rustamkutovyi) [reply src/modules/user/_test/userDeleteById.spec.js:47]: Done

File: src/modules/user/_test/userDeleteById.spec.js (hunk 1 above)
Comment (rustamkutovyi) [reply src/modules/user/_test/userDeleteById.spec.js:25]: Done

**Review by itmelia — APPROVED** [review]

**Pull request merged**